pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -193,6 +195,43 @@ func validateSystemRequirements(mountUtil mount.Interface) (features, error) {
 	return f, nil
 }
 
+func validateSwapConfiguration(fs fs.FS, failSwapOn bool) error {
```

Comment:
Is this intended to be in a PR about adding tests? It looks like this is new functionality in container_manager-linux.

Reply:
I see now you factored it so you can use it in the tests.